includes/frontend/class-wcmp-checkout.php

```diff
@@ -410,6 +410,15 @@ public static function save_delivery_options($order_id)
                 WCMYPA_Admin::META_DELIVERY_OPTIONS,
                 $deliveryOptions->toArray()
             );
+
+            /**
+             * Save meta deliverydate for filter in ordergrid
```

Comment:
`Save delivery date in meta for use as order grid filter.`

Reply:
done